osq-exts/tables/crowdstrikefalconagent/crowdstrikefalconagent_darwin.go

```diff
@@ -0,0 +1,205 @@
+package crowdstrikefalconagent
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+
+	"github.com/groob/plist"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const (
```

Comment:
Might be nice to slurp up `grouping-tags` also (although I believe this would require an additional shell-out as I don't believe `falconctl stats` returns defined `grouping-tags`)?

Also, I am curious how y'all determined what data (from `stats` output) is valuable vs what is not valuable? I don't have any qualms with your choices, just curious since not all data output by `stats` is being defined here (for example, `smcache` attribute values are not included in this `const`).

Reply:
I can add more data in, but also wanted to be mindful that all three operating systems have different data/values available.  If a user doesn't want a particular set, then the query can be modified to omit that column.  I think the idea here is to have as much data available as possible, in case someone wants to do a SELECT *, but the user also has the option just to query a portion of the data, like SELECT agentID from CSFA, etc.
That being said, I can go back in and add in the additional grouping-tags, but request it would be a later PR.
Also, I think smcache might be a new value added by stats recently.  the "falconctl stats --plist" returns a subset of data of "falconctl stats" output 8(.

Reply:
I've put in requests to make stats have parity with --plist output.  Its on a list somewhere.